### ISSUE TITLE
Where to find backend

### ISSUE THREAD

**Issue by iammikev** [vlinx-io/NativeImageAnalyzer#3]
Hello, in order to execute native-image-analyzer we need to connect to a backend running on localhost.
Is this backend provided somewhere?

**Comment by vlinx-io** [comment]
Hi @SnifflePiffle , sorry, we do not provide backend for now.

**Comment by molenzwiebel** [comment]
What is the use of this tool if we cannot run our own backend, but the tool connects to `localhost:16003`?

**Comment by vlinx-io** [comment]
I'm sorry, it seems that we built the application from the wrong branch. In fact, we have provided a remote backend and will rebuild the application.


What we need to explain is that this is just a test project for the purpose of verifying feasibility. Many functions are not yet perfect and only support decompiling the main function. At the same time, we cannot guarantee the long-term availability of this service.
